phpstan-baseline.neon

```diff
@@ -0,0 +1,10 @@
+parameters:
+	ignoreErrors:
+	    -
+	        message: "#^Parameter \\#2 \\$eventDispatcher of method Pimcore\\\\Bundle\\\\PersonalizationBundle\\\\Targeting\\\\Condition\\\\EventDispatchingConditionInterface\\:\\:postMatch\\(\\) expects Symfony\\\\Component\\\\EventDispatcher\\\\EventDispatcherInterface, Symfony\\\\Contracts\\\\EventDispatcher\\\\EventDispatcherInterface given\\.$#"
+	        count: 1
+	        path: src/Targeting/ConditionMatcher.php
+	    -
+	        message: "#^Parameter \\#2 \\$eventDispatcher of method Pimcore\\\\Bundle\\\\PersonalizationBundle\\\\Targeting\\\\Condition\\\\EventDispatchingConditionInterface\\:\\:preMatch\\(\\) expects Symfony\\\\Component\\\\EventDispatcher\\\\EventDispatcherInterface, Symfony\\\\Contracts\\\\EventDispatcher\\\\EventDispatcherInterface given\\.$#"
```

Comment:
The interface with namespace `Components` just extends the one with namespace `Contracts` and if we change the parameter type to expect `Contracts`, it would support both and be compatible, right?


See also https://github.com/symfony/event-dispatcher/blob/04046f35fd7d72f9646e721fc2ecb8f9c67d3339/EventDispatcherInterface.php#L14-L23

Reply:
Yes, Done in [4ab1cbd](https://github.com/pimcore/personalization-bundle/pull/24/commits/4ab1cbdf5663eebfd38c9644a29219b4f5494708)

Reply:
nice! 

I would say that we keep the baseline file with only
```
parameters:
	ignoreErrors:
```

Reply:
Done in [09609a0](https://github.com/pimcore/personalization-bundle/pull/24/commits/09609a0b8fb5effa60bc8259cc76298be98ad166)